### DISCUSSION TITLE
Smarty 5 demo

### DISCUSSION THREAD

**Discussion by sutrus** [smarty-php/smarty#842] [Q&A] [answered]
Hi,
I'm trying to test the Smarty5 demo project.
Composer installed everything in the vendor folder without any problems.
I added require_once '../vendor/autoload.php' to the file in index.php;
Is this version functional or am I doing something wrong?
Test on Wampserver, php 7.4, 8.0, 8.1

The result ends with an error 
`Fatal error: Class "Smarty\Parser\TemplateParser" not found in C:\wamp64\www\smarty-smarty5\src\Compiler\Template.php on line 1272`

**Comment by wisskid** [comment, marked as answer]
I'm afraid Smarty v5 is not ready yet. I hope to publish a release candidate for testing very soon.

**Reply by danjessen** [reply]
Is there a place to follow the development? I'm just asking because there are 3 websites now and one subreddit. I have a somewhat hard time of follow the development or development path. But I might be looking in the wrong places ?

**Reply by wisskid** [reply]
The best place to look for code, releases, issues and discussions is here on Github. The docs are published [here](https://smarty-php.github.io/smarty/4.x/). I'm currently working on rewriting the documentation for v4 and v5.

**Reply by danjessen** [reply]
Interesting. I still prefer the old docs. But I see the need to update

**Comment by wisskid** [comment]
You could try running `make` from the smarty dir where Makefile is though. That should create the missing files.

**Comment by sutrus** [comment]
Thank you for the info. I'll check it out later.

**Comment by marclaporte** [comment]
See discussions at https://github.com/smarty-php/smarty/pull/852

**Comment by marclaporte** [comment]
Smarty 5.0 has been released: https://github.com/smarty-php/smarty/discussions/956

We can close this.
